trackintel/model/trips.py

```diff
@@ -28,11 +31,11 @@ class TripsAccessor(object):
     (e.g., waiting) between two relevant activities.
 
     The following assumptions are implemented
-
-        - All movement before the first and after the last activity is omitted
         - If we do not record a person for more than `gap_threshold` minutes, we assume that the person performed an \
             activity in the recording gap and split the trip at the gap.
-        - Trips that start/end in a recording gap can have an unknown origin/destination.
+        - Trips that start/end in a recording gap can have an unknown origin/destination staypoint.
+        - If the origin (or destination) staypoint is unknown (and a geometry column exists), the origin/destination
+            geometry is set as the first coordinate of the first tripleg (or the last coordinate of the last tripleg)
```

Comment:
[Doc]
"geometry" needs to start exactly below "if" 
https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#lists-and-quote-like-blocks